### ISSUE TITLE
Copilot panel cannot load

### ISSUE THREAD

**Issue by jarvan-jiang** [MarkMpn/Sql4Cds#538]
tried both v9.3 and v9.2, none of them works. XrmToolbox  version is v1.2023.12.68.
After successful setup copilot settings, the panel still cannot load like this:
![image](https://github.com/user-attachments/assets/2d3a4bc8-6874-4b41-abd5-197c75389fa5)

**Comment by MarkMpn** [comment]
Can I confirm you don't see the "Enter your question here" box at the bottom (your screenshot only shows the top of the panel which I'd expect to be blank), and you don't get any visible error messages? Can you also check if you get any relevant messages in the log file at `%appdata%\MscrmTools\XrmToolBox\Logs\MarkMpn.Sql4Cds.XTB.log`?

**Comment by jarvan-jiang** [comment]
> Can I confirm you don't see the "Enter your question here" box at the bottom (your screenshot only shows the top of the panel which I'd expect to be blank), and you don't get any visible error messages? Can you also check if you get any relevant messages in the log file at `%appdata%\MscrmTools\XrmToolBox\Logs\MarkMpn.Sql4Cds.XTB.log`?

Yeah, the bottom is blank too, and there is no error messages, initially the panel is not visible, after I set up copilot setting it auto pop out, and keep blank with no other messages.

And there is no log in that file.

**Comment by MarkMpn** [comment]
The Copilot panel uses the WebView2 control - if you're running on a device where this isn't already installed as part of Windows you may need to install it yourself from https://developer.microsoft.com/en-gb/microsoft-edge/webview2?form=MA13LH#download-section
